docs/docusaurus/src/theme/Admonition/Types.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import DefaultAdmonitionTypes from '@theme-original/Admonition/Types';
+
+function CtaAdmonition(props) {
+    return (
+        <div>
+            <h3>{props.title}</h3>
+            <div>{props.children}</div>
+        </div>
```

Comment:
I think we can replace the container with a `section` tag instead.
```suggestion
        <section>
            <h3>{props.title}</h3>
            <div>{props.children}</div>
        </section>
```